[2021-07-02 14:27:42,129] {taskinstance.py:876} INFO - Dependencies all met for <TaskInstance: get_data_patient_and_diseases.getting_tasks.get_diseases 2021-07-01T00:00:00+00:00 [queued]>
[2021-07-02 14:27:42,166] {taskinstance.py:876} INFO - Dependencies all met for <TaskInstance: get_data_patient_and_diseases.getting_tasks.get_diseases 2021-07-01T00:00:00+00:00 [queued]>
[2021-07-02 14:27:42,166] {taskinstance.py:1067} INFO - 
--------------------------------------------------------------------------------
[2021-07-02 14:27:42,166] {taskinstance.py:1068} INFO - Starting attempt 1 of 1
[2021-07-02 14:27:42,166] {taskinstance.py:1069} INFO - 
--------------------------------------------------------------------------------
[2021-07-02 14:27:42,184] {taskinstance.py:1087} INFO - Executing <Task(PythonOperator): getting_tasks.get_diseases> on 2021-07-01T00:00:00+00:00
[2021-07-02 14:27:42,191] {standard_task_runner.py:52} INFO - Started process 5731 to run task
[2021-07-02 14:27:42,206] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'get_data_patient_and_diseases', 'getting_tasks.get_diseases', '2021-07-01T00:00:00+00:00', '--job-id', '1070', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/get_Data_Patient.py', '--cfg-path', '/tmp/tmprhh66320', '--error-file', '/tmp/tmp9wpxt1rd']
[2021-07-02 14:27:42,207] {standard_task_runner.py:77} INFO - Job 1070: Subtask getting_tasks.get_diseases
[2021-07-02 14:27:42,408] {logging_mixin.py:104} INFO - Running <TaskInstance: get_data_patient_and_diseases.getting_tasks.get_diseases 2021-07-01T00:00:00+00:00 [running]> on host airflowvm
[2021-07-02 14:27:42,568] {taskinstance.py:1280} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=get_data_patient_and_diseases
AIRFLOW_CTX_TASK_ID=getting_tasks.get_diseases
AIRFLOW_CTX_EXECUTION_DATE=2021-07-01T00:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2021-07-01T00:00:00+00:00
[2021-07-02 14:27:42,569] {python.py:151} INFO - Done. Returned value was: None
[2021-07-02 14:27:42,599] {taskinstance.py:1184} INFO - Marking task as SUCCESS. dag_id=get_data_patient_and_diseases, task_id=getting_tasks.get_diseases, execution_date=20210701T000000, start_date=20210702T142742, end_date=20210702T142742
[2021-07-02 14:27:42,670] {taskinstance.py:1245} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2021-07-02 14:27:42,697] {local_task_job.py:151} INFO - Task exited with return code 0
